vtStor/DriveManager.cpp

```diff
@@ -43,17 +62,23 @@ eErrorCode cDriveManager::EnumerateDrives( eScanForHardwareChanges ScanForHardwa
     {
         // scan for hardware changes
         m_Drives.clear();
+        m_DevicePaths.clear();
     }
+    vtStor::GetStorageDevicePaths(m_DevicePaths, eOnErrorBehavior::Continue);
 
     U32 count = 0;
-    for ( auto& enumerator : m_DriveEnumerators )
+
+    for (const auto& devicePath : m_DevicePaths)
     {
-        error = enumerator->EnumerateDrives(m_Drives, count);
-        if (eErrorCode::None != error)
+        for (auto& enumerator : m_DriveEnumerators)
         {
-            //TODO: handle error
+            error = enumerator->EnumerateDrive(devicePath, m_Drives, count);
```

Comment:
How does this prevent more than one enumerator from enumerating the same device?